test/Table_Tests/src/In_Memory/Split_Tokenize_Spec.enso

```diff
@@ -0,0 +1,171 @@
+from Standard.Base import all
+
+import Standard.Base.Data.Text.Case_Sensitivity.Case_Sensitivity
+import Standard.Test.Extensions
+
+from Standard.Table import Table
+from Standard.Table.Errors import Invalid_Value_Type, Column_Count_Exceeded, Duplicate_Output_Column_Names, Missing_Input_Columns
+from Standard.Test import Test, Test_Suite, Problems
+
+spec =
+    tables_equal t0 t1 =
+        same_headers = (t0.columns.map .name) == (t1.columns.map .name)
+        same_columns = (t0.columns.map .to_vector) == (t1.columns.map .to_vector)
+        same_headers && same_columns
+    tables_should_be_equal actual expected =
+        equal = tables_equal actual expected
+        if equal.not then
+            msg = 'Tables differ.\nActual:\n' + actual.display + '\nExpected:\n' + expected.display
+            Test.fail msg
```

Comment:
This is already implemented by a `should_equal` override in `Util.enso`.

I'd suggest to use _that_. But I really like the `actual.display` change - I think it could be incorporated into that method.